docs/index.md

```diff
@@ -27,9 +27,9 @@ toc: false
 #### **Code Chaos**:
 Ending the cycle of repetitive code creation, ensuring longevity through structured development, and preventing code morphing with each developer's touch.
 #### **Tight Coupling**:
-Minimizing dependencies by using flexible containers and allowing easy addition or removal of components. This ensures smoother integration and maintenance of the system.
+Minimizing reliance on third-party dependencies by providing its own unique tools for managing multiple dependencies, enabling easy addition or removal of components. Ensuring smoother integration and maintenance of the system.
```

Comment:
The problem is right coupling with a single dependency. 

mnimizing reliance on a single thirst-party dependency that provide all the tools.  core-lib provide tool to work multiple dependencies the are easy to add or remove. Ensuring smoother integration and maintenance of the system.